docs/src/DeveloperGuide/ProfileTransformationCpp.md

```diff
@@ -62,11 +68,18 @@ The pattern defined in this snippet matches IR like:
   %2 = load %Qubit*, %Qubit** %1, align 8
 ```
 
-In the above rule, the first and a second argument of `__quantum__rt__array_get_element_ptr_1d` is captured as `arrayName` and `index`, respectively. Likewise, the bitcast instruction is captured as `cast`. Each of these captures will be available inside the replacement function `access_replacer`.
+In the above rule, the first and a second argument of
+`__quantum__rt__array_get_element_ptr_1d` is captured as `arrayName` and
+`index`, respectively. Likewise, the bitcast instruction is captured as `cast`.
+Each of these captures will be available inside the replacement function
+`access_replacer`.
 
 ## Implementing replacement logic
 
-After a positive match is found, the lead instruction alongside a IRBuilder, a capture table and a replacement table is passed to the replacement function. Here is an example on how one can access the captured variables to perform a transformation of the IR:
+After a positive match is found, the lead instruction alongside a IRBuilder, a
+capture table and a replacement table is passed to the replacement function.
+Here is an example on how one can access the captured variables to perform a
+transformation of the IR:
 
 ```c++
```

Comment:
is `c++` or `cpp` preferred in the code samples. Both are being used.

Reply:
Yes, you are right. There are two places with `cpp` and the rest with `c++`. However, I believe that hljs uses `cpp` in the YAML file as to gives nicer filenames while it seems to be indifferent to which of the two are used in the docs. I'd suggest to use `cpp` in the YAML file and `c++` for all code blocks. If you prefer otherwise, let me know.